gradio.egg-info/PKG-INFO

```diff
@@ -1,11 +1,14 @@
-Metadata-Version: 1.0
+Metadata-Version: 2.1
 Name: gradio
 Version: 2.7.0
 Summary: Python library for easily interacting with trained machine learning models
 Home-page: https://github.com/gradio-app/gradio-UI
 Author: Abubakar Abid, Ali Abid, Ali Abdalla, Dawood Khan, Ahsen Khaliq
 Author-email: team@gradio.app
 License: Apache License 2.0
-Description: UNKNOWN
 Keywords: machine learning,visualization,reproducibility
 Platform: UNKNOWN
+License-File: LICENSE
+
+UNKNOWN
+
```

Comment:
I do not know what these changes are

Reply:
@pngwn the egg file is created when you install the gradio package, it is probably coming with setup-tools;

I think you can remove egg related files from the PR

[Ref](https://www.python101.pythonlibrary.org/chapter38_eggs.html)

Reply:
Thanks!